corehq/apps/users/models.py

```diff
@@ -588,16 +588,20 @@ class _AuthorizableMixin(IsMemberOfMixin):
         Use either SingleMembershipMixin or MultiMembershipMixin instead of this
     """
 
-    def get_domain_membership(self, domain):
+    def get_domain_membership(self, domain, allow_mirroring=True):
         domain_membership = None
         try:
             for d in self.domain_memberships:
                 if d.domain == domain:
                     domain_membership = d
                     if domain not in self.domains:
                         raise self.Inconsistent("Domain '%s' is in domain_memberships but not domains" % domain)
-            if not domain_membership and domain in self.domains:
-                raise self.Inconsistent("Domain '%s' is in domain but not in domain_memberships" % domain)
+            if not domain_membership:
+                if domain in self.domains:
+                    raise self.Inconsistent("Domain '%s' is in domain but not in domain_memberships" % domain)
```

Comment:
nit: this sentence is really confusing. what does `Domain is in domain` mean?

Reply:
`MultiMembershipMixin` has a `domains` property and a `domains_membership` property that are apparently supposed to stay in sync. I think we mostly use `domains_membership` and don't know exactly what the `domains` list is for.

Reply:
does that mean the second `domain` should be plural? That is clearly a minor thing, but it does seem like a typo that makes this more confusing.